.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
Could you add a comment here saying that sudo is required for installing softhsm please? Then it'll be obvious that we can revert this if/when we no longer need to use SoftHSMv1.